backend/report.js

```diff
@@ -294,6 +304,11 @@ function createProjection(filterObj) {
     if(filterObj.isCommentNeeded) {
         projection.comments = '$comments';
     }
+    if(filterObj.isCommentNeeded) {
+        projection.readyForApprove = '$readyForApprove';
+        projection.size = {$size:'$readyForApprove'};
+    }
+    console.log(filterObj);
```

Comment:
Do not use console.log in production code. Do you need it? Or it's debug output?